cloud/google/gce.py

```diff
@@ -502,6 +502,59 @@ def create_instances(module, gce, instance_names):
 
     return (changed, instance_json_data, instance_names)
 
+def start_instances(module, gce, instance_names, zone_name):
```

Comment:
Think we can get it into one function?  The two functions are kinda boilerplate.  I don't see a proliferation of states anytime soon but one function would be easier to maintain.